aviary/variable_info/variable_meta_data.py

```diff
@@ -2248,6 +2260,19 @@
     default_value=0.0,
 )
 
+add_meta_data(
+    Aircraft.Engine.SHAFT_POWER_DESIGN,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
```

Comment:
possible similar variable:
INPROP.HPMSLS = max sea level static horsepower, hp

Reply:
I added this to the variable_meta_data